performanceplatform/collector/piwik/base.py

```diff
@@ -0,0 +1,29 @@
+from performanceplatform.utils import requests_with_backoff
```

Comment:
Actually looking at it, this is probably ok - it may be best to rename `query.py` to `core.py` to make more consistent with ga / pingdom - again webtrends is inconsistent here.

I'd like to know @tombooth and @jcbashdown s thoughts on this as well